source/Dig/base.util.persongenerator.bmx

```diff
@@ -928,111 +934,252 @@ End Type
 
 
 '=== GREEK ===
-Type TPersonGeneratorCountry_Greek extends TPersonGeneratorCountry
+rem
```

Comment:
DIese Änderung in den anderen PR verschieben und hier wirklich nur den polnischen Personengenerator ergänzen?

Reply:
Ich raeume den dann hier nochmal auf bzw mache einen sauberen neuen auf...wenn der griechen-branch uebernommen wurde.